### ISSUE TITLE
Upgrade to 0.25.0

### ISSUE THREAD

**Issue by imsnif** [libpnet/libpnet#397]
Hey @JuxhinDB - possibly related to the conversation here: https://github.com/imsnif/bandwhich/issues/101

I'm trying (make a PR) to upgrade `libpnet` on https://github.com/hughesac/packet_builder and not really managing.

The farthest I've got is with these dependencies:
```
[dependencies]
derive-new = "0.5"
ipnetwork = "0.12.8"
pnet = "0.25.0"
pnet_datalink = "0.25.0"
pnet_base = "0.22.0"
pnet_macros_support = "0.25.0"
```
At which point when I do `cargo test` I get:
```
⋊> ~/c/packet_builder on master ⨯ cargo test                                                                                                                                                                                           22:43:01
   Compiling packet-builder v0.4.0 (/home/aram/code/packet_builder)
    Finished dev [unoptimized + debuginfo] target(s) in 0.83s
     Running target/debug/deps/packet_builder-eff724336d4d9e8c

running 7 tests
test arp::tests::macro_arp_basic ... ok
test ethernet::tests::macro_ether_basic ... ok
test icmp::tests::macro_icmp_basic ... ok
test tcp::tests::macro_tcp_basic ... ok
test udp::tests::macro_udp_basic ... ok
test vlan::tests::macro_vlan_basic ... ok
test ipv4::tests::macro_ipv4_basic ... FAILED

failures:

---- ipv4::tests::macro_ipv4_basic stdout ----
thread 'ipv4::tests::macro_ipv4_basic' panicked at 'assertion failed: `(left == right)`
  left: `0`,
 right: `1`: Cannot sum mis-aligned words at 0x7fb7f5ed4fe7', /home/aram/.cargo/registry/src/github.com-1ecc6299db9ec823/pnet_packet-0.25.0/src/util.rs:71:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.


failures:
    ipv4::tests::macro_ipv4_basic

test result: FAILED. 6 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--lib'
```

Do you have a suggestion for a direction?

**Comment by JuxhinDB** [comment]
Hey @imsnif -- this is a new one.. I'll be honest I only tried building, but the test suites seemed to be passing during the version dump. You're hitting this bug consistently?

**Comment by JuxhinDB** [comment]
OK seems to be a PR I merged that caused the breaking change - https://github.com/libpnet/libpnet/issues/399

Will see what I can do to fix it.

**Comment by pheki** [comment]
Hasn't this been fixed by #403?

**Comment by kishiguro** [comment]
I'm expecting this would be fixed by pnet-0.26.0 update as well.

**Comment by kishiguro** [comment]
@imsnif would you mind to try 0.26.0 and see how it goes?

**Comment by imsnif** [comment]
Hey @hughesac - do you think you can try upgrading packet_builder to 0.26.0? I think it'll be faster on your end. If it works, I think I'll be able to upgrade bandwhich to use the new version of packet_builder and then also upgrade its own dependency on pnet to 0.26.0. What say you?

**Comment by hughesac** [comment]
Thanks for the heads up.  I upgraded to 0.26.0 and the build and tests are working again.  The version with the changes is 0.5.0.

**Comment by kishiguro** [comment]
Thanks for confirming!  let me close this.ticket.

**Comment by imsnif** [comment]
Hey, just released bandwhich 0.14.0 that depends on pnet 0.26.0 and packet_builder 0.5.0. Looks great. Thanks to the both of you!
